hphp/hack/src/errors/errors.ml

```diff
@@ -4750,9 +4750,15 @@ let override_no_default_typeconst pos_child pos_parent =
 let inout_annotation_missing pos1 pos2 =
   let msg1 = (pos1, "This argument should be annotated with `inout`") in
   let msg2 = (pos2, "Because this is an `inout` parameter") in
-  add_list (Typing.err_code Typing.InoutAnnotationMissing) msg1 [msg2]
-
-let inout_annotation_unexpected pos1 pos2 pos2_is_variadic =
+  let start_column = Pos.start_cnum pos1 in
```

Comment:
Turns out that `cnum` is 'character number', a file offset, not a column number. Argh. This seems to work:

```suggestion
  let (_, start_column) = Pos.line_column pos1 in
```